test/e2e/manifest_test.go

```diff
@@ -272,6 +272,55 @@ var _ = Describe("Podman manifest", func() {
 			))
 	})
 
+	It("authenticated push", func() {
+		if podmanTest.Host.Arch == "ppc64le" {
+			Skip("No registry image for ppc64le")
+		}
+		session := podmanTest.Podman([]string{"manifest", "create", "foo"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+		session = podmanTest.Podman([]string{"pull", ALPINE})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+
+		lock := GetPortLock("5000")
+		defer lock.Unlock()
+
+		session = podmanTest.Podman([]string{"run", "-d", "-p", "5000:5000", "--name", "registry", "-e", "REGISTRY_AUTH=htpasswd", "-e",
```

Comment:
Note to @containers/podman-maintainers:

We should migrate the tests to be using `github.com/containers/podman/v4/hack/podman-registry-go`: https://github.com/containers/podman/blob/main/pkg/bindings/test/auth_test.go#L10